src/containers/my-resources/add-resource-modal/AddResourceModal.tsx

```diff
@@ -94,17 +97,29 @@ const AddResourceModal = <T extends TableItem>({
         {...props}
       />
 
-      <Box sx={styles.buttonsArea}>
-        <AppButton onClick={closeModal} variant={ButtonVariantEnum.Tonal}>
-          {t('common.cancel')}
-        </AppButton>
-        <AppButton
-          disabled={!selectedRows.length}
-          onClick={onAddItems}
-          sx={styles.addButton}
-        >
-          {t('common.add')}
-        </AppButton>
+      <Box sx={showCheckboxWithTooltip ? styles.formControls : {}}>
```

Comment:
I think would be better like this `showCheckboxWithTooltip && styles.formControls`

Reply:
Thanks for the suggestion! I initially considered using `showCheckboxWithTooltip && styles.formControls`, but I opted for the conditional approach (`<Box sx={showCheckboxWithTooltip ? styles.formControls : {}}>`) to ensure that the `sx` prop always receives a valid value, even when `showCheckboxWithTooltip `is `false`. In that case, it returns an empty object `{}` instead of `false`, which avoids potential type issues with Material-UI's `Box` component that expects a value of type `SxProps<Theme> | undefined` :
<img width="990" alt="Screenshot 2024-08-20 at 13 24 46" src="https://github.com/user-attachments/assets/264222de-d6d4-4310-9e11-d539b9bd9063">